src/subprocess-posix.cc

```diff
@@ -93,6 +91,9 @@ bool Subprocess::Start(SubprocessSet* set, const string& command) {
     // In the console case, output_pipe is still inherited by the child and
     // closed when the subprocess finishes, which then notifies ninja.
   }
+#ifdef POSIX_SPAWN_USEVFORK
```

Comment:
I think on linux with glibc one is to define `__USE_GNU` before including spawn.h to get this define. But then we either have to unconditionally define that, or check for glibc.

I wasn't able to measure a performance difference when I wrote the TODO, and in theory C libraries could be smart (if their are no atfork handlers, etc, then use vfork for posix_spawn automatically). On several systems, they are already smart.

So unless there's data that shows that this measurably helps somewhere, I'm a bit skeptical about this change.